### PR TITLE
FEATURE: support QueryTickers API on bybit

### PR DESCRIPTION
**Pull request by bailantaotao**

### CODE REVIEW

File: pkg/exchange/bybit/exchange.go [hunk 1]
```diff
@@ -49,6 +50,7 @@ func (e *Exchange) PlatformFeeCurrency() string {
 func (e *Exchange) QueryMarkets(ctx context.Context) (types.MarketMap, error) {
 	instruments, err := e.client.NewGetInstrumentsInfoRequest().Do(ctx)
 	if err != nil {
+		log.Warnf("Failed to query instruments, err: %v", err)
```
Comment (c9s) [inline pkg/exchange/bybit/exchange.go:53]: logs start with lower case

File: pkg/exchange/bybit/exchange.go [hunk 2]
```diff
@@ -59,3 +61,47 @@ func (e *Exchange) QueryMarkets(ctx context.Context) (types.MarketMap, error) {
 
 	return marketMap, nil
 }
+
+func (e *Exchange) QueryTicker(ctx context.Context, symbol string) (*types.Ticker, error) {
+	s, err := e.client.NewGetTickersRequest().Symbol(symbol).DoWithResponseTime(ctx)
+	if err != nil {
+		log.Warnf("Failed to get tickers, symbol: %s, err: %v", symbol, err)
```
Comment (c9s) [inline pkg/exchange/bybit/exchange.go:68]: ditto

File: pkg/exchange/bybit/bybitapi/get_tickers_request.go [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+		return nil, err
+	}
+
+	var data Tickers
+	if err := json.Unmarshal(resp.Result, &data); err != nil {
+		return nil, err
+	}
+
+	// assign response timestamp to Tickers
+	data.Time = resp.Time
```
Comment (gx578007) [inline pkg/exchange/bybit/bybitapi/get_tickers_request.go:69]: This looks like local time. Could you add more comments about the reason why using response time?

File: pkg/exchange/bybit/exchange.go [hunk 4]
```diff
@@ -59,3 +61,47 @@ func (e *Exchange) QueryMarkets(ctx context.Context) (types.MarketMap, error) {
...
+	}
+
+	ticker := toGlobalTicker(s.List[0], s.Time.Time())
+	return &ticker, nil
+}
+
+func (e *Exchange) QueryTickers(ctx context.Context, symbols ...string) (map[string]types.Ticker, error) {
+	tickers := map[string]types.Ticker{}
+	if len(symbols) > 0 {
+		for _, s := range symbols {
```
Comment (gx578007) [inline pkg/exchange/bybit/exchange.go:84]: Is it required to sleep between api calls?

**Review by gx578007 — APPROVED** [review]
Others LGTM

File: pkg/exchange/bybit/exchange.go [hunk 5]
```diff
@@ -59,3 +61,47 @@ func (e *Exchange) QueryMarkets(ctx context.Context) (types.MarketMap, error) {
...
+			t, err := e.QueryTicker(ctx, s)
```
Comment (c9s) [inline pkg/exchange/bybit/exchange.go:85]: This might hit the rate limit issue

File: pkg/exchange/bybit/exchange.go (hunk 4 above)
Comment (bailantaotao) [reply pkg/exchange/bybit/exchange.go:103]: ok, i will implement him

File: pkg/exchange/bybit/bybitapi/get_tickers_request.go (hunk 3 above)
Comment (bailantaotao) [reply pkg/exchange/bybit/bybitapi/get_tickers_request.go]: OK

**Review by c9s — APPROVED** [review]

**Pull request merged**